app/src/main/java/com/example/cinemapp/ui/main/movie_details/CastAdapter.kt

```diff
@@ -36,8 +40,15 @@ class CastAdapter : RecyclerView.Adapter<CastAdapter.CastViewHolder>() {
 
     override fun onBindViewHolder(holder: CastViewHolder, position: Int) {
         holder.bind(cast[position])
-        if (position == 0)
-                (holder.binding.cvCardPerson.layoutParams as ViewGroup.MarginLayoutParams).marginStart += 10
+        if (position == 0) {
```

Comment:
Seems like this logic is duplicated (See CreditsAdapter). try to extract this logic into some kind of utility method